src/lib.rs

```diff
@@ -158,12 +158,35 @@ impl CertPaths {
         }
 
         let mut certs = match &self.file {
-            Some(cert_file) => load_pem_certs(cert_file)?,
+            Some(cert_file) => match load_pem_certs(cert_file) {
```

Comment:
Do you think it's worth abstracting over this to reduce duplication a little bit?

Reply:
:+1: I pushed a revision that lifts out the error construction. It was helpful, turns out I could also simplify the test a little bit too.